api_backend.yaml

```diff
@@ -645,84 +645,150 @@ paths:
           description: Unavailable service
           schema:
             $ref: "#/definitions/ProblemJson"
+
+  "/user-data-processing":
+    x-swagger-router-controller: UserDataProcessingController
+    post:
+      operationId: upsertUserDataProcessing
+      summary: Set User's data processing choices
+      description: Upsert user data processing for the current authenticated user.
```

Comment:
can you change this with the actual use case (something like "Let the authenticated user express his will to retrieve or delete his stored data"